app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+from typing import List, Union
```

Comment:
There is a syntax error in the import statement. The correct syntax should be `from typing import List, Union` without the slash (`/`). This will cause a syntax error and needs to be fixed.